src/plugins/LADTable/components/LadRow.vue

```diff
@@ -260,7 +260,7 @@ export default {
         event.preventDefault();
         this.preview(this.objectPath);
       } else {
-        const resultUrl = identifierToString(this.openmct, this.objectPath);
+        const resultUrl = objectPathToUrl(this.openmct, this.objectPath);
```

Comment:
nice catch

Reply:
right! we had a default export in addition to named exports in that file... scary